app/src/main/java/com/example/trempelapp/domain_layer/WriteTokenUseCaseImpl.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.trempelapp.domain_layer
+
+import com.example.trempelapp.data_layer.AuthRepository
+import javax.inject.Inject
+
+class WriteTokenUseCaseImpl @Inject constructor(
+    private val authRepository: AuthRepository
+) : UseCase<String, String> {
+
+    override fun execute(params: String): String {
```

Comment:
why are you returning string here?

Reply:
in case you want just to write token, use Unit parameter and you can miss your return parameter

Reply:
done